src/main/java/randoop/contract/CompareToAntiSym.java

```diff
@@ -0,0 +1,67 @@
+package randoop.contract;
+
+import randoop.Globals;
+
+/**
+ * The contract: Reversing a compareTo on two objects should result in equal values of opposite signs
```

Comment:
I suggest explictly using the term "antisymmetric" somewhere in the Javadoc, especially since an abbreviated version of that word is in the class name.